service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -78,5 +82,19 @@
     @NotNull(message = "You need to add details to children")
     @Valid
     private final List<@NotNull(message = "You need to add details to children")Element<Child>> children1;
+    @NotBlank(message = "Enter Familyman case number", groups = NoticeOfProceedingsGroup.class)
+    private final String familyManCaseNumber;
+    private List<ProceedingType> proceedingTypes;
+
+    public List<Element<Applicant>> getAllApplicants() {
+        return applicants != null ? applicants : new ArrayList<>();
+    }
+
+    public List<Element<Child>> getAllChildren() {
+        return children1 != null ? children1 : new ArrayList<>();
+    }
+
+    @NotNull(message = "Enter hearing details", groups = NoticeOfProceedingsGroup.class)
     private final List<Element<HearingBooking>> hearingDetails;
+    private final List<Element<DocumentBundle>> noticeOfProceedingsBundle;
```

Comment:
Should the list name not match the variable name ?

Reply:
No I want this to be a bit more re-usable incase we need it in the future. Not overall keen on 'DocumentBundle'

Reply:
👍